_data/investing.yml

```diff
@@ -202,6 +202,8 @@ websites:
       sms: Yes
       phone: Yes
       hardware: Yes
+      exceptions:
+        text: "Only support: Yubikey FIDO U2F Security Key, Yubikey NEO, Yubikey 4 NANO, Yubikey 4"
```

Comment:
`Yubikey FIDO U2F Security Key` Seems like they just call it "Security Key" so you could probably compress it to just `Yubikey Security Key`.
You don't list any Yubikeys from the 5 series. Are they not supported?
What's the underlying technology that's needed in the keys to login?
If it's U2F you could just say `"Hardware requires a Yubikey with U2F support"`

Reply:
The fidelity site specifically lists those 4 devices only.

I've tried others, and it will not accept the keys, therefore not utilizing true universal U2F.

Reply:
Just got a reply from them on Twitter. Turns out they don't support the 5 series. Only Yubikey 4